src/vs/workbench/browser/parts/views/viewPaneContainer.ts

```diff
@@ -344,7 +345,8 @@ export class ViewPaneContainer extends Component implements IViewPaneContainer {
 	}
 
 	getTitle(): string {
-		let title = Registry.as<ViewletRegistry>(Extensions.Viewlets).getViewlet(this.getId()).name;
+		const composite = Registry.as<ViewletRegistry>(ViewletExtensions.Viewlets).getViewlet(this.getId()) || Registry.as<PanelRegistry>(PanelExtensions.Panels).getPanel(this.getId());
```

Comment:
Looks like `ViewContainer` should support `name` property. Can you please create a follow up item for this?